### PR TITLE
Update deps and fix new linting rules

### PR DESCRIPTION
**Pull request by sethvargo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,39 +29,38 @@
...
     "@actions/exec": "^1.1.0",
     "@actions/tool-cache": "^1.7.1",
-    "@google-github-actions/setup-cloud-sdk": "^0.1.1",
-    "@lifeomic/attempt": "^3.0.0",
-    "@types/tmp": "^0.2.1",
-    "fs": "0.0.2",
-    "googleapis": "^91.0.0",
-    "google-auth-library": "^7.10.1",
+    "@google-github-actions/setup-cloud-sdk": "^0.3.0",
+    "@lifeomic/attempt": "^3.0.1",
```
Comment (bharathkkb) [inline package.json:36]: I think some of these are no longer used.

File: src/deploy-cloudrun.ts [hunk 2]
```diff
@@ -45,8 +45,7 @@ export async function run(): Promise<void> {
     const source = core.getInput('source'); // Source directory
     const suffix = core.getInput('suffix');
     const tag = core.getInput('tag');
-    const noTraffic =
-      core.getInput('no_traffic').toLowerCase() == 'true' ? true : false;
+    const noTraffic = core.getInput('no_traffic').toLowerCase() == 'true' ? true : false;
```
Comment (bharathkkb) [inline src/deploy-cloudrun.ts:48]: nit: we can switch to `getBooleanInput`

**Review by bharathkkb — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (sethvargo) [reply package.json]: Removed. Also moved a few to devDeps, since they are only used int ests

File: src/deploy-cloudrun.ts (hunk 2 above)
Comment (sethvargo) [reply src/deploy-cloudrun.ts]: Done

**Pull request merged**